From fa1789ad786f3b31693143bd82ac440540819472 Mon Sep 17 00:00:00 2001
From: harlan <hmccanne@exclamationpoint.com>
Date: Wed, 11 Feb 2026 23:08:53 +0000
Subject: [PATCH] Fix compile errors

---
 src/power_control.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/power_control.cpp b/src/power_control.cpp
index 39a2dd59d4..eef56f4c7b 100644
--- a/src/power_control.cpp
+++ b/src/power_control.cpp
@@ -39,6 +39,7 @@
 #include <string_view>
 
 static std::shared_ptr<sdbusplus::asio::dbus_interface> taskIface;
+static std::shared_ptr<sdbusplus::asio::dbus_interface> osSysIface;
 
 namespace power_control
 {
@@ -659,9 +660,9 @@ static void setPowerState(const PowerState state)
     {
         HostState = true;
     }
-    if ((powerState == power_control::PowerState::on) ||
+    if ((powerState == power_control::PowerState::on) || (
         (powerState == power_control::PowerState::cycleOff) &&
-            (HostState == true))
+            (HostState == true)) )
     {
         try
         {
